### ISSUE TITLE
Basic pagination problem when deleting posts

### ISSUE THREAD

**Issue by githubertus** [opensource-socialnetwork/opensource-socialnetwork#1357]
To reproduce:

- switch off Autopagination
- create 11 newsfeed wall posts numbered from no.1 to no.11 
- reload newsfeed page

You'll see 10 entries, starting with no.11 on top, down to no.2 at the bottom
Your paginator looks like First | 1 | 2 | Last

- delete posting no.11

Since deleting a post currently doesn't trigger a re-calculation of the paginator, it still looks like
 First | 1 | 2 | Last
- click on paginator tab 2, and you'll see ... nothing.

Of course not, because meanwhile posting no.1 has become a member of offset=1 

In other words: the same number of posts you have deleted on the current page, the same number of posts will be missing from the next page.

Because Autopagination is based on that old logic, it's doing not better. Besides that you'll get an endless spinning load icon in the special case (like my example above) if there was only 1 post on the last page.

**Comment by lianglee** [comment]
perhaps we load the page after delete? because i don't see other solution here.

**Comment by githubertus** [comment]
Just had a weird idea: For experimental purposes I was often using a component named "customPagination". I currently installed it on 5b like
```
function custom_pagination_init(){
	ossn_add_hook("pagination", "page_limit","set_new_page_limit");
}
function set_new_page_limit(){
   return 1;
}
```

I just tested: This is working flawlessly with Autopagination, too.

Ok, so maybe the following would be possible on submitting a post delete:
- before actually deleting the record from db: switch page limit temporarely to 1
- fake a page scroll (which would load only the next post as a result then, old offset still in use)
- switch back page limit to default
- finally delete record from db

**Comment by lianglee** [comment]
Maybe we can do following : 

     IF OFFSET > 1 AND NO RESULT THEN REDIRECT (home)

**Comment by Lablnet** [comment]
.

**Comment by githubertus** [comment]
have sent you some experimental code on skype ...

**Comment by githubertus** [comment]
fixed by #1363
